src/backend/common/helpers/prediction_helper.py

```diff
@@ -371,6 +371,42 @@ def calculate_before_match(self, i: int) -> TComputedMatchInfo:
                     count = 0
                     count = score_breakdown[color]["totalChargeStationPoints"]
                     means[color] = count
+                elif self._stat == "note_scored":
+                    count = 0
+                    for mode in ["auto", "teleop"]:
+                        for position in ["Amp", "Speaker"]:
+                            if position == "Speaker":
+                                for type in ["NoteCount", "NoteAmplifiedCount"]:
+                                    count += score_breakdown[color][
+                                        "{}{}{}".format(mode, position, type)
```

Comment:
`autoSpeakerNoteAmplifiedAcount` isn't a valid breakdown key

Reply:
should be fixed in commit [71f1a41](https://github.com/the-blue-alliance/the-blue-alliance/pull/5746#:~:text=existant%20breakdown%20key-,71f1a41,-Merge%20state)